manifests/setup/prometheus-operator-deployment.yaml

```diff
@@ -40,6 +40,18 @@ spec:
             memory: 100Mi
         securityContext:
           allowPrivilegeEscalation: false
+      - args:
+        - --logtostderr
+        - --secure-listen-address=:8443
+        - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+        - --upstream=http://127.0.0.1:8080/
```

Comment:
can prometheus-operator just listen on `127.0.0.1`?

Reply:
I didn't see such option in prometheus-operator CLI flags.

Reply:
let's create an issue then upstream, and potentially add it there.

Reply:
Issue: https://github.com/coreos/prometheus-operator/issues/3095